sdk/src/auth.rs

```diff
@@ -63,10 +70,11 @@ impl NetworkAuth {
     }
 
     /// Signs a message to submit a proof.
```

Comment:
docs: Signs a message to mark a proof as ready for proof generation.

Reply:
added